libbpf-rs/src/program.rs

```diff
@@ -419,4 +432,8 @@ impl Program {
     pub fn insn_cnt(&self) -> usize {
         unsafe { libbpf_sys::bpf_program__insn_cnt(self.ptr) as usize }
     }
+
+    pub fn unload(&mut self) {
```

Comment:
@insearchoflosttime can you please drop unload (libbpf doesn't expose it anymore), add doc comments and resubmit?